### PR TITLE
Update support.md

### PR DESCRIPTION
**Pull request by karenstrother**
Added note that partners should not publish contact center phone numbers per [LG-11912](https://cm-jira.usa.gov/browse/LG-11912)

### CODE REVIEW

File: _pages/support.md [hunk 1]
```diff
@@ -554,7 +554,7 @@ There are some cases where our team is unable to help, typically either for secu
   </h3>
   <div id="user-support" class="usa-accordion__container">
 <div class="usa-accordion__content" markdown="1">
-If you receive a complaint from a user having trouble logging in or managing their account, it is best to direct them to [submit a help ticket to user support](https://www.login.gov/contact/).
+If you receive a complaint from a user having trouble logging in or managing their account, it is best to direct them to the Login.gov help center at https://www.login.gov/help/. Do not publish Login.gov contact center phone numbers.
```
Comment (Sgtpluck) [inline _pages/support.md:557]: do we want the url to be hyperlinked?

**Review by Sgtpluck — APPROVED** [review]
Had a question, but looks good

File: _pages/support.md (hunk 1 above)
Comment (karenstrother) [reply _pages/support.md:557]: Thank you, yes.

Comment (karenstrother) [reply _pages/support.md]: Fixed

**Pull request merged**